arch/x86_64/include/intel64/arch.h

```diff
@@ -275,7 +277,7 @@
 #  define MSR_X2APIC_ICR_OTHERS        0x000c0000  /* Send to all APICs, excluding self. */
 #  define MSR_X2APIC_ICR_BUSY          0x00001000
 #  define MSR_X2APIC_ICR_FIXED         0x00000000
-#  define MSR_X2APIC_DESTINATION(d)    ((d) << 32)
+#  define MSR_X2APIC_DESTINATION(d)    ((d) << 32ul)
```

Comment:
please remove this change, it was fixed by a8ec8b8cd291e6edd7be269cb02b420925588d49